src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlStream.cs

```diff
@@ -324,39 +284,21 @@ internal SqlCachedStream(SqlCachedBuffer sqlBuf)
             _cachedBytes = sqlBuf.CachedBytes;
         }
 
-        override public bool CanRead
-        {
-            get
-            {
-                return true;
-            }
-        }
+        public override bool CanRead => true;
 
-        override public bool CanSeek
-        {
-            get
-            {
-                return true;
-            }
-        }
+        public override bool CanSeek => true;
 
-        override public bool CanWrite
+        public override bool CanWrite
```

Comment:
This can be change to use expression syntax as well can't it?

Reply:
@Wraith2 Yes that has been updated along with few more possible body expression changes. Thanks.